docs/software/SDK/Baballonia Overview.mdx

```diff
@@ -0,0 +1,98 @@
+---
+title: Overview of Baballonia SDK
+sidebar_position: 1
+---
+
+# Overview of Baballonia SDK
+
+The Baballonia SDK provides developers with the tools and interfaces necessary to integrate eye-tracking and face-tracking capabilities into their applications. This SDK is designed to work seamlessly with the Baballonia software, allowing for easy access to processed tracking data and raw camera streams.
+
+It enables developers to build custom plugins, create new capture sources, and develop integrations inside the Baballonia application—without needing to modify Baballonia’s core codebase itself.
+
+---
+
+## SDK Components
+
+### **1. Capture System (Camera Stream Behavior)**
+
+At the core of the SDK is the Capture System: a unified API for accessing frames from any supported camera source. Rather than tying Baballonia to a single device type, each camera is represented as a subclass of the abstract `Capture` class.
+
+:::info The `Capture` Base Class
+- Defines how a camera source starts and stops streaming  
+- Handles thread-safe delivery of OpenCV `Mat` frame objects  
+- Tracks device readiness and connection status  
+- Allows developers to safely produce and consume frames across multiple threads  
+:::
+
+This makes it possible for Baballonia to support:
+
+- Standard USB webcams (UVC)  
+- Varjo eye-tracking cameras  
+- Android phone camera streams  
+- V4L2 devices on Linux  
+- Custom or experimental capture sources  
+
+<h3><b><i>All through the same interface.</i></b></h3>
+
+---
+
+### **2. Capture Factories (Plugin System)**
+
+The SDK includes the `ICaptureFactory` interface, which allows developers to create custom camera capture sources as plugins. By implementing this interface, developers can define how their specific camera hardware integrates with the Baballonia application.
```

Comment:
It would make more sense to introduce the `Capture` and `ICaptureFactory` together - one cannot exist without the other.